php/fm-overlays-singleton.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ * fm-overlays-post-type.php
```

Comment:
copy-pasta. but also, this file should be named `class-fm-overlays-singleton.php`